x-pack/libbeat/persistentcache/store_test.go

```diff
@@ -16,18 +15,16 @@ import (
 )
 
 func TestStandaloneStore(t *testing.T) {
-	type valueType struct {
-		Something string
-	}
 
 	var key = []byte("somekey")
 	var value = []byte("somevalue")
 
-	tempDir, err := ioutil.TempDir("", "beat-data-dir-")
+	tempDir, err := os.MkdirTemp("", "beat-data-dir-")
```

Comment:
Let's use `t.TempDir` instead so we clean up after the test too.

```suggestion
	tempDir, err := t.TempDir()
```